### PR TITLE
[FIX] profiler: Fix psql log path

### PR DESCRIPTION
**Pull request by hugho-ad**

### CODE REVIEW

File: profiler/controllers/main.py [hunk 1]
```diff
@@ -143,6 +143,8 @@ def dump_pgbadger(self, dir_dump, output, cursor):
         filename = os.path.join(dir_dump, output)
         pg_version = dump_db_manifest(cursor)['pg_version']
         log_path = os.environ.get('PG_LOG_PATH', DFTL_LOG_PATH % pg_version)
+        if not os.path.isfile(log_path):
+            log_path = DFTL_LOG_PATH % pg_version
```
Comment (moylop260) [inline profiler/controllers/main.py:147]: What is different to line 145?
Maybe you need delete the environment variable

Comment (hugho-ad) [reply profiler/controllers/main.py:147]: Some how, it is returning `/var/log/pg_log` (I need to debug), that is a directory not a file, so I just verify that the path is a file not a dir

Comment (hugho-ad) [reply profiler/controllers/main.py:147]: what do you think  @moylop260 ?

Comment (moylop260) [reply profiler/controllers/main.py:147]: That is because `PG_LOG_PATH` has a wrong value.
You need change it from it was assigned

Comment (hugho-ad) [reply profiler/controllers/main.py:147]: That is the point, `PG_LOG_PATH` env var has not any value, 
I make sure when printing the env var `echo ${PG_LOG_PATH}` it returns empty, so DFTL_LOG_PATH is used, some how when is getted like this `os.environ.get('PG_LOG_PATH', DFTL_LOG_PATH % pg_version)` it returns `/var/log/pg_log`

Comment (moylop260) [reply profiler/controllers/main.py:147]: If `DFTL_LOG_PATH` is used:
 - https://github.com/Vauxoo/odoo-profiler/blob/2f13a7116514e6dc38cb4426663ca679474fd43b/profiler/controllers/main.py#L24

So, you should have a returns like as: `/var/lib/postgresql/9.3/main/pg_log/postgresql.log` and not `/var/log/pg_log`
Could you use a pdb and send us a print-screen using a print of `os.environ.get('PG_LOG_PATH')` and a print of `DFTL_LOG_PATH % pg_version` please?
Maybe there is a sub-script assigning the variable

**Comment by moylop260** [comment]
Fixed from https://github.com/Vauxoo/docker-ubuntu-base/pull/40

**Pull request closed without merging**